eve/main.yml

```diff
@@ -166,6 +166,29 @@ stages:
         env:
           <<: *global_env
     - ShellCommandWithSecrets: *test_zenko
+    - ShellCommandWithSecrets:
+        name: Upgrade Zenko
+        command: >-
+          helm upgrade $ZENKO_HELM_RELEASE charts/zenko
+          --namespace %(prop:testNamespace)s
+          -f eve/ci-values.yml
+          -f eve/ft-values.yml
+          --timeout 200
+          --wait
+        haltOnFailure: true
+        env:
+          <<: *global_env
+    - ShellCommand:
+        name: Wait for cloudserver
+        command: sleep 30
+    - ShellCommandWithSecrets:
+        name: Zenko Functional Tests
+        command: |-
+          CLOUDSERVER_POD=$(kubectl get pods --no-headers=true -o custom-columns=:metadata.name | grep cloudserver-manager)
+          kubectl exec $CLOUDSERVER_POD -- bash -c "npm run install_ft_deps"
```

Comment:
There's no way you guys can apply the same "workflow" that you use for other tests?
```
building a test image -> push to registry -> run with proper commands to test stuff. 
```
I'm curious to know if this is what is causing the liveness probe to fail. (Note that I might be super wrong here as I don't know much about cloudserver :) )

Reply:
Yes, that'll be better 🙂